src/inc/quic_driver_main.h

```diff
@@ -0,0 +1,37 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    QUIC Perf Main common driver code.
+
+--*/
+
+#pragma once
+
+#include "quic_platform.h"
+
+#define QUIC_RUN_SUCCESS 0
+#define QUIC_RUN_MISSING_TEST_TYPE -1
+#define QUIC_RUN_FAILED_QUIC_OPEN -2
+#define QUIC_RUN_FAILED_TEST_INITIALIZE -3
+#define QUIC_RUN_MISSING_NECESSARY_ARGUMENTS -4
+#define QUIC_RUN_UNKNOWN_TEST_TYPE -5
+#define QUIC_RUN_INVALID_MODE -6
+#define QUIC_RUN_STOP_FAILURE -7
```

Comment:
Why define your error codes and not just use QUIC_STATUS?

Reply:
Because theres not a QUIC_STATUS that maps to most of these. Most would just be QUIC_STATUS_INTERNAL_ERROR. I can switch them over though if thats cleaner.